tdms/include/openandorder.h

```diff
@@ -0,0 +1,12 @@
+#include "matio.h"
```

Comment:
For consistency, snake case this filename? At a glance I parsed the name as "operand order", so snake case would avoid that and match the naming convention with other files.